cdk/lib/mparticle-api.ts

```diff
@@ -98,16 +117,15 @@ export class MParticleApi extends GuStack {
 			}),
 		);
 
-		// Allow the lambda to assume the roles that allow cross-account fetching of tags
-		// lambda.addToRolePolicy(
-		// 	new PolicyStatement({
-		// 		actions: ['sts:AssumeRole'],
-		// 		effect: Effect.ALLOW,
-		// 		resources: [
-		// 			// mobileAccountRoleArn.valueAsString,
-		// 			// targetingAccountRoleArn.valueAsString,
-		// 		],
-		// 	}),
-		// );
+		new SrLambdaDomain(this, {
+			subdomain: 'mparticle-api',
+			restApi: apiGateway.api,
+		});
+
+		new CfnOutput(this, 'MParticleLambdaRoleArn', {
+			value: mparticleLambdaRole.roleArn,
```

Comment:
ah now I see why you made an explicit role - but the default should be available like this
```suggestion
			value: lambda.role.roleArn,
```

Reply:
Changed @johnduffell , thank you.